src/ZfcRbac/Guard/AbstractGuard.php

```diff
@@ -68,18 +68,13 @@ public function onResult(MvcEvent $event)
             403
         ));
 
-        $event->stopPropagation(true);
-
         $application  = $event->getApplication();
         $eventManager = $application->getEventManager();
 
-        if (method_exists($eventManager, 'triggerEvent')) {
-            // ZF3 EventManager
-            $event->setName(MvcEvent::EVENT_DISPATCH_ERROR);
-            $eventManager->triggerEvent($event);
-        } else {
-            // ZF2 EventManager
-            $eventManager->trigger(MvcEvent::EVENT_DISPATCH_ERROR, $event);
-        }
+        $event->setName(MvcEvent::EVENT_DISPATCH_ERROR);
+        $eventManager->triggerEvent($event);
```

Comment:
just in case someone will come and complain about BC break. Yes this can be BC break because we depend on `EventManagerInterface` and if someone crazy enough implement it themselves and not use EventManager this is BC break but do we care? :)